### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by Mar1nka**

### CODE REVIEW

**Comment by IharKrasnik** [comment]
@SobakaSlava review please

File: src/client/components/error/error.jsx [hunk 1]
```diff
@@ -10,7 +11,7 @@ const formatError = (err) => {
   if (err instanceof ApiError) {
     if (err.serverError) {
       return defaultMessage;
-    } else if (err._status === 400) {
+    } if (err._status === 400) {
```
Comment (SobakaSlava) [inline src/client/components/error/error.jsx]: Please place if on a new line

File: .eslintrc.js [hunk 2]
```diff
@@ -11,5 +14,6 @@ module.exports = {
         "**/*.config.js",
       ],
     }],
+    "react/jsx-one-expression-per-line": ["warn"],
```
Comment (SobakaSlava) [inline .eslintrc.js]: What is the reason of adding this line?

File: src/client/pages/signup/signup.jsx [hunk 3]
```diff
@@ -83,11 +101,13 @@ export default class Signup extends PureComponent {
...
         <Layout.HeaderContent state={states.green}>
           <Auth className={styles.panel}>
-            {this.state.signupSuccess ? (
+            {signupSuccess ? (
               <div className={styles['signup-success']}>
                 <h2>Thank you for signing up!</h2>
                 <p>
-                  The verification email has been sent to {this.state.email}. <br />
+                  The verification email has been sent to
+                  { email }.
```
Comment (SobakaSlava) [inline src/client/pages/signup/signup.jsx]: It's better not to use spaces in curly braces when using string literals (and in other places of this PR)

**Comment by IharKrasnik** [comment]
@Mar1nka please check comments from @SobakaSlava

File: .eslintrc.js (hunk 2 above)
Comment (Mar1nka) [reply .eslintrc.js]:
```
<p>
                  The verification email has been sent to
                  {email}.
                  <br />
                  Please follow the instructions from the email to complete a signup process.
                </p>

Comment (Mar1nka) [reply .eslintrc.js]: `{email}.`
To be able to put a dot here, and not to transfer it to a new line (in this case, a space will be added before the dot, this is undesirable)

Comment (IharKrasnik) [reply .eslintrc.js]: @SobakaSlava that's correct. We decided to use warn for now unless you see a better solution. By default as @Mar1nka mentioned it forces to put {email} on the separate line which adds a space before the dot

**Pull request merged**